config/webpack.config.js

```diff
@@ -21,6 +21,7 @@ const entryFiles = {
   '1995-edu-benefits': './src/js/edu-benefits/1995/edu-benefits-entry.jsx',
   '5490-edu-benefits': './src/js/edu-benefits/5490/edu-benefits-entry.jsx',
   '5495-edu-benefits': './src/js/edu-benefits/5495/edu-benefits-entry.jsx',
+  '526EZ-claims-increase': './src/js/526EZ/form-entry.jsx',
```

Comment:
I think this path is missing `disability-benefits`

Reply:
Woops....good catch!